torch/_inductor/compile_fx.py

```diff
@@ -186,10 +187,12 @@ def clone_preserve_strides(x):
 
 
 def align_inputs(model, inputs, static_input_idxs=()):
+    def is_aligned(storage_offset, dtype):
+        return (storage_offset * get_dtype_size(dtype)) % ALIGNMENT == 0
```

Comment:
The test here is incomplete as we don't have the guard on the allocator yet, would be good yo have a todo at least (and a debug assert in paranoid mode that storage pointers are aligned)